styles/styles.css

```diff
@@ -368,20 +370,32 @@ main .section .default-content-wrapper picture img {
 }
 
 .button-container a {
+  background-color: transparent;
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+  cursor: pointer;
   display: inline-block;
-  padding: .75em 1.5em;
   font-family: var(--font-family-proxima);
```

Comment:
Is this still Proxima?

Reply:
I was planning on updating the fonts in a separate pull request, and then updating all `--font-family-proxima` to use the new font, and changing the variable name to `--font-family-primary`

I can do that in this PR if you would prefer though.